pkgs/os-specific/linux/kernel/hardened-config.nix

```diff
@@ -19,10 +19,10 @@ ${optionalString (versionAtLeast version "4.11") ''
   GCC_PLUGIN_STRUCTLEAK y # A port of the PaX structleak plugin
```

Comment:
Please take out the changes to the hardened config, if it is not to be used with the grsec kernel.